### ISSUE TITLE
Question about training & eval

### ISSUE THREAD

**Issue by HenryHZY** [TencentARC/ST-LLM#4]
Hi @farewellthree @yxgeee @xinntao @yeliudev @ARCer , thanks for your great project.

I have reproduced the training and encounter pretty low acc in the MVBench eval.

1.Training
I follow the training Instruction (config & dataset) and execute the official training scripts:
```
config="instructblipbase_stllm_qa"
mkdir -p output/$config
deepspeed --master_port=20000 --include=localhost:0,1,2,3,4,5,6,7 \
    stllm/train/train_hf.py \
    --cfg-path config/$config.yaml
```

The data structure of the training output is as follows:
```
output/instructblipbase_stllm_qa
├── checkpoint-6194
│   ├── ...
│   ├── ...
├── config.json
├── generation_config.json
├── pytorch_model.bin [~15GB]
├── trainer_state.json
└── training_args.bin
```

And here is my training loss, which **seems very normal**. Could you please check if the final loss is ~0.5?
<img width="1430" alt="image" src="https://github.com/TencentARC/ST-LLM/assets/43438692/e56b5949-7533-49c7-a27f-b1270939c1c1">

2.Eval
However, when I try to use the trained model (**output/instructblipbase_stllm_qa**) with the official eval script, I encounter pretty low acc in the MVBench eval (acc<10%). 

Is that anything wrong in my eval scripts? Or do I need to modify something (e.g., config) for the trained model?
```
python stllm/test/mvbench/mv_bench_infer.py \
    --cfg-path config/instructblipbase_stllm_qa.yaml \
    --ckpt-path output/instructblipbase_stllm_qa \
    --anno-path /path/to/mvbench/json \
    --output_dir $output \
    --output_name instructblipbase_stllm_qa_mvbench_fps1 \
    --num-frames 0 \
    --ask_simple
```

By the way, I have used the official QA model checkpoint for MVBench eval and obtain approximate result as the paper. Thank you very much for your assistance!

**Comment by HenryHZY** [comment]
Currently there is no "pytorch_model.bin.index.json" or "tokenizer*" in the trained model path (output/instructblipbase_stllm_qa). I think there may exist some bugs in the model saving.

**Comment by HenryHZY** [comment]
After discussing with the author, I fix my issue by replacing "--ckpt-path output/instructblipbase_stllm_qa" with "--ckpt-path output/instructblipbase_stllm_qa/pytorch_model.bin".

**Comment by ruishuzhao** [comment]
Hi， @HenryHZY 
After modifying the model path,
What about the performance of your trained model on the MVbench？
And how many epochs you used？
